.github/workflows/bumpversion.yml

```diff
@@ -18,4 +18,4 @@ jobs:
       - name: Create bump and changelog
         uses: commitizen-tools/commitizen-action@master
         with:
-          github_token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
```
If you use GITHUB_TOKEN instead of PERSONAL_ACCESS_TOKEN, the job won't trigger another workflow. It's like using [skip ci] in other CI's.
```

Do you know if that is solved? If not, here is explained how to create the personal: https://commitizen-tools.github.io/commitizen/tutorials/github_actions/

Reply:
Thanks. I have no clue about it. I will follow the guideline